doc/cephfs/nfs.rst

```diff
@@ -2,8 +2,10 @@
 NFS
 ===
 
-CephFS namespaces can be exported over NFS protocol using the
-`NFS-Ganesha NFS server`_.
+CephFS namespaces can be exported over NFS protocol using the `NFS-Ganesha NFS
+server`_. The simplest and preferred way of managing nfs-ganesha clusters and
+cephfs exports is using ``ceph nfs ...`` commands. See :doc:`/mgr/nfs` for more
+details.
```

Comment:
While you're here, explain what the remainder of this doc is for.

Reply:
Good point! I think we need more details on why `ceph nfs ` commands are the simplest and preferred? I think this doc is for folks who want to manage the deployment, life cycle, and high availability of NFS-ganesha servers using their preferred mechanism instead of relying on ceph-adm/rook to do so?  What do you think?

Reply:
I have updated the doc